ref-exchange/src/account_deposit.rs

```diff
@@ -5,26 +5,74 @@ use std::collections::HashMap;
 use near_contract_standards::fungible_token::core_impl::ext_fungible_token;
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::{ValidAccountId, U128};
-use near_sdk::{assert_one_yocto, env, near_bindgen, AccountId, Balance, PromiseResult};
+use near_sdk::{
+    assert_one_yocto, env, near_bindgen, 
+    AccountId, Balance, PromiseResult, StorageUsage,
+};
 
 use crate::utils::{ext_self, GAS_FOR_FT_TRANSFER};
 use crate::*;
 
 // [AUDIT_01]
-const MAX_ACCOUNT_LENGTH: u128 = 64;
-const MAX_ACCOUNT_BYTES: u128 = MAX_ACCOUNT_LENGTH + 4;
-const MIN_ACCOUNT_DEPOSIT_LENGTH: u128 = 1 + MAX_ACCOUNT_BYTES + 16 + 4;
+const U128_STORAGE: StorageUsage = 16;
+const U64_STORAGE: StorageUsage = 8;
+const U32_STORAGE: StorageUsage = 4;
+/// max length of account id is 64 bytes. We charge per byte.
+const ACC_ID_STORAGE: StorageUsage = 64;
+/// As a key, 4 bytes length would be added to the head
+const ACC_ID_AS_KEY_STORAGE: StorageUsage = ACC_ID_STORAGE + 4;
+/// As a near_sdk::collection key, 1 byte prefiex would be added to the head, right?
+const ACC_ID_AS_CLT_KEY_STORAGE: StorageUsage = ACC_ID_AS_KEY_STORAGE + 1;
```

Comment:
u32 is used for length of strings, so it's always will be 4.

Reply:
This 1 means compare to a key in lookupmap (ACC_ID_AS_CLT_KEY_STORAGE) with a key in hashmap (ACC_ID_AS_KEY_STORAGE),   , they all have a U32 length of the key after sirealize and the only diff is a prefix byte is added to the one in lookupmap, right? So, using increment by 1 to describe that diff.